src/System.Management.Automation/engine/remoting/common/RunspaceConnectionInfo.cs

```diff
@@ -3350,7 +3365,23 @@ private void GetContainerPropertiesInternal()
 
                     var computeSystemPropertiesHandle = getComputeSystemPropertiesInfo.Invoke(null, new object[] { ComputeSystem });
 
-                    RuntimeId = (Guid)computeSystemPropertiesType.GetProperty("RuntimeId").GetValue(computeSystemPropertiesHandle);
+                    // Since Hyper-V changed this from a property to a field, we can optimize for newest Windows to see if it's a field,
+                    // otherwise we fall back to old code to be compatible with older versions of Windows
+                    var fieldInfo = computeSystemPropertiesType.GetField("RuntimeId");
+                    if (fieldInfo != null)
+                    {
+                        RuntimeId = (Guid)fieldInfo.GetValue(computeSystemPropertiesHandle);
+                    }
+                    else
+                    {
+                        var propertyInfo = computeSystemPropertiesType.GetProperty("RuntimeId");
+                        if (propertyInfo == null)
+                        {
+                            throw new PSInvalidOperationException(StringUtil.Format(RemotingErrorIdStrings.CannotGetHostInteropTypes));
```

Comment:
> StringUtil.Format(RemotingErrorIdStrings.CannotGetHostInteropTypes)

Since there's no argument, `StringUtil.Format` is not needed. Same to other instances of this error message above.

Reply:
Fixed all instances in this file